controller/cmd/destination/main.go

```diff
@@ -58,6 +64,17 @@ func Main(args []string) {
 		log.Warnf("expected cluster domain through args (falling back to %s)", *clusterDomain)
 	}
 
+	// Set opaqueports.DefaultOpaquePorts variiable
+	ports, err := parsePorts(*opaquePorts)
```

Comment:
Could we figure out a way to use the parsing functions in `util`'s `parsing.go` here? We already have some separate functions for parsing opaque ports spread around that package as well as in `server.go`, and it feels unfortunate to introduce another place where we are parsing opaque ports.

Reply:
Makes sense. Moved this into `util`